From 3af64ea2d2a0b76da1a974966d9be21779e308e8 Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Fri, 25 Mar 2011 09:49:15 -0400
Subject: [PATCH] NetworkStatus: fix vpn indicator

It looks like a part of the code wasn't updated to reflect
a variable name change.
---
 js/ui/status/network.js |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/js/ui/status/network.js b/js/ui/status/network.js
index 7990332..4bfbd21 100644
--- a/js/ui/status/network.js
+++ b/js/ui/status/network.js
@@ -1752,7 +1752,7 @@ NMApplet.prototype = {
                         }
                     }
                 } else
-                    a._primaryDevice = this._vpnDevice;
+                    a._primaryDevice = this._devices.vpn.device
 
                 if (a._primaryDevice)
                     a._primaryDevice.setActiveConnection(a);
-- 
1.7.4.1

